### DISCUSSION TITLE
Usage of KHR extensions

### DISCUSSION THREAD

**Discussion by suro7** [google/model-viewer#4148] [Q&A]
Hello.
Is it possible to enable/disable certain KHR extension while model-viewer is running (runtime) or on the load?
Particularly KHR Unlit is very important for us.

### Version
- model-viewer: v1.9.1

### Browser Affected
- [ ] Chrome, version: 110.0.5481.178

### OS
- [ ] Windows 11

Looking forward hearing from you soon
Suren

**Comment by elalish** [comment]
We already support all the KHR extensions, including Unlit. They are enabled by including them in your GLB - no need to configure MV.

**Reply by suro7** [reply]
The model already include KHR unlit. 
The question is whether it's possible to turn KHR unlit extension on and off during loading of the model or dynamically after loading.

**Reply by elalish** [reply]
No, it's not currently. Why would you want to turn it on and off? I would think it would be easier to simply load a new GLB, especially since you'd likely need different textures and such.
